pyproject.toml

```diff
@@ -64,7 +64,12 @@ soap-bpnn = [
     "wigners",
 ]
 pet = [
-  "pet @ git+https://github.com/lab-cosmo/pet@5d40710",
+    "pet-neighbors-convert @ git+https://github.com/lab-cosmo/pet-neighbors-convert.git",
```

Comment:
Unfortunately we will need to find a way to release this extension on PyPI as well... it won't be easy at all